Source/CsDebugScript.Engine/IBreakpoint.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+
+namespace CsDebugScript
+{
+    /// <summary>
+    /// Status of action execution on breakpoint hit.
+    /// </summary>
+    public enum OnBreakpointHit
```

Comment:
>OnBreakpointHit [](start = 16, length = 15)

Name sounds more like event and not like enumeration.
Maybe it should be renamed to: BreakpointHitActionStatus? BreakpointHitStatus?

Reply:
BreakpointHitResult?

---
In reply to: [195943811](https://github.com/southpolenator/WinDbgCs/pull/16#discussion_r195943811) [](ancestors = 195943811)

Reply:
Done